modules/rostests/apitests/wininet/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 
-add_executable(wininet_apitest InternetOpen.c testlist.c)
+add_executable(wininet_apitest Download.c InternetOpen.c testlist.c)
 target_link_libraries(wininet_apitest wine)
 set_module_type(wininet_apitest win32cui)
 #add_delay_importlibs(wininet_apitest wininet)
-add_importlibs(wininet_apitest msvcrt kernel32 ntdll)
+add_importlibs(wininet_apitest wininet msvcrt kernel32 ntdll)
```

Comment:
The lack of this import was not an oversight. Adding it invalidates several tests in InternetOpen.c